runtime/compiler/control/J9Options.cpp

```diff
@@ -1202,7 +1202,13 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
 
 void J9::Options::preProcessMmf(J9JavaVM *vm, J9JITConfig *jitConfig)
    {
+#if defined(TR_TARGET_64BIT) && defined(J9ZOS390)
+   PORT_ACCESS_FROM_JAVAVM(vm);
+   OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+#endif
```

Comment:
Best to just move this down to where it's actually used, so we don't have two separate ifdefs.

Reply:
I have moved the changes to the below ifdef.

Reply:
Warnings aren't enabled on z/OS yet; the internal build succeeded.